chaco/abstract_mapper.py

```diff
@@ -49,13 +49,6 @@ def map_data_array(self, screen_vals):
     # ------------------------------------------------------------------------
     # Persistence-related methods
     # ------------------------------------------------------------------------
-    def __getstate__(self):
-        state = super(AbstractMapper, self).__getstate__()
-        for key in ["_cache_valid"]:
```

Comment:
`_cache_valid` is not a trait on this class or any of its base classes

Reply:
sigh. but it is a trait on the subclasses e.g. `Base1DMapper`

Reply:
Ah you are right.  TBH I believe that all related traits should be transient even if they weren't perviously involved in a `__getstate__`method.  For example other classes have `_screen_cache_valid` or `_selection_cache_valid` traits.  I think I will update all of these to be transient as I can't see why they shouldn't all be so